### ISSUE TITLE
DT Implement County Map Page

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/FreeBMD2#341]
https://www.freebmd.org.uk/county-map.html	

See #296 for implementation suggestions - less monlithic.

For discussion - for MVP a simple page very much as at current, but enhancement as per #296 in a later phase.

**Comment by richardofsussex** [comment]
If you look on test3, pages such as https://test3.freebmd.org.uk/districts/county_summary?county_code=GLS are already implemented. However, for some reason, the same URL pattern gives an error on beta. @Vino-S to investigate/remedy, please.

**Comment by richardofsussex** [comment]
Fixed - now on test3.

**Comment by KristinaGadzhieva** [comment]
agreed to remove 'all counties' from a drop-down list

**Comment by KristinaGadzhieva** [comment]
SG feedback: to make a list of districts for all counties (similar to FreeBMD 1)

**Comment by richardofsussex** [comment]
See #613 - I think Jeff's suggestion relates to searching, not the County Map page.

**Comment by richardofsussex** [comment]
See e.g. https://beta.freebmd.org.uk/districts/county_summary?county_code=HAM&button=. I think this issue can be closed.

**Comment by KristinaGadzhieva** [comment]
@richardofsussex Agree, but still we need to remove 'all counties' from a drop-down list.

**Comment by richardofsussex** [comment]
Is this the one? :-<img width="1000" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNEtPQ0E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--90336e068bb63e959fc592eb0473a81aefbe93de/image.png" alt="image.png" />

**Comment by richardofsussex** [comment]
If so, now on beta.
